src/game.py

```diff
@@ -133,12 +124,7 @@ def infect_city(self, city, colour):
         logging.info(
             f'Infecting {infected_city} with {colour} disease.')
         if infected_city.cubes[colour] < 3:
-            if self.disease_cubes[colour] == 0:
-                raise NullDiseaseCapacityException(colour)
-            self.disease_cubes[colour] -= 1
-            logging.debug(
-                (f'{colour} disease capacity is now '
-                 f'{self.disease_cubes[colour]}.'))
+            self.diseases[colour].decrease_resistance(1) # 1 cube
```

Comment:
No needs to leave a comment here - integer parameter already looks self-explaining.
(if it wasn't, we would have to add one more well-named variable).